init.ps1

```diff
@@ -57,7 +61,17 @@ function AppendContent($Message, $Uri, $BuildScriptPath, [switch]$Enable, [hasht
     if($enable){
         Write $Message
         $progressPreference = 'silentlyContinue'
-        $content = (Invoke-WebRequest -Uri $Uri -UseBasicParsing -Headers @{"Cache-Control"="no-cache"} ).content
+		$requestResult = (Invoke-WebRequest -Uri $Uri -UseBasicParsing -Headers @{"Cache-Control"="no-cache"} )

```

Comment:
I ran unto a problem where my Powershell IDE, PowerGUI made the ps1 file be treated as binary.  When the script was ran in the IDE there was not problems.  When I used the Invoke-WebRequest the script failed since it's mime type was a stream rather than text.  I wanted to be able to use the same Invoke-WebRequest way to execute it so I know it would work correctly when ran through the WebRequest.  It turned out that the IDE was saving the file with a binary header.  When I saved it using Textpad++ it saved with the correct meta header.